governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md

```diff
@@ -0,0 +1,83 @@
+---
+title: SparkLend Ethereum - Collateral Parameter Changes - February 26, 2024
+summary: Signal your support or opposition to the SparkLend Collateral Parameter Changes
+discussion_link: https://forum.makerdao.com/t/feb-22-2024-proposed-changes-to-sparklend-for-upcoming-spell/23739
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-02-26T16:00:00
+end_date: 2024-02-29T16:00:00
+---
+# Poll: SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, February 26 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following [SparkLend Ethereum](https://app.spark.fi/markets/?marketName=proto_spark_v3) Collateral Parameter Changes.
+
+#### WETH
```

Comment:
[This table](https://makerdao-forum-backup.s3.dualstack.us-east-1.amazonaws.com/original/3X/6/f/6fe6d0181f8494eee4aa9f6ace857ba91c6b73b0.png) shows other parameters have changed, too. Not sure why those were left out later in the original post